addons/website/static/tests/tours/focus_blur_snippets.js

```diff
@@ -0,0 +1,79 @@
+odoo.define(function (require) {
+'use strict';
+
+const ajax = require('web.ajax');
+const tour = require('web_tour.tour');
+
+const blockIDToData = {
+    parent: {
+        selector: '.s_focusblur',
+        name: 'section',
+        number: 0,
+    },
+    child1: {
+        selector: '.s_focusblur_child1',
+        name: 'first child',
+        number: 1,
+    },
+    child2: {
+        selector: '.s_focusblur_child2',
+        name: 'second child',
+        number: 2,
+    },
+};
+
+function clickAndCheck(blockID, expected) {
+    const blockData = blockIDToData[blockID] || {};
+
+    return [{
+        trigger: blockData.selector || '#wrapwrap',
+        content: blockID ? `Enable the ${blockData.name}` : 'Disable all blocks',
+    }, {
+        trigger: blockID
+            ? `.oe_overlay.ui-draggable:eq(${blockData.number}).oe_active`
+            : `#oe_manipulators:not(:has(.oe_active))`,
+        content: 'Once the related overlays are enabled/disabled, check that the focus/blur calls have been correct.',
+        run: function (actions) {
+            const result = window.focusBlurSnippetsResult;
+            window.focusBlurSnippetsResult = [];
+
+            if (expected.length !== result.length
+                    || !expected.every((item, i) => item === result[i])) {
+                console.error(`
+                    Expected: ${expected.toString()}
+                    Result: ${result.toString()}
+                `);
+            }
+        },
+    }];
+}
+
+window.focusBlurSnippetsResult = [];
+
+tour.register('focus_blur_snippets', {
+    url: '/?enable_editor=1',
+}, [
+    {
+        trigger: 'body',
```

Comment:
I'd always add a content key so it is easier to find/grep when there is an error (I even thought it was mandatory)

Reply:
Yep, ok (even though if there is an error on the first step, it's easy enough to find ^^ And there should be no error finding the body anyway ^^)